Apps/W1/ConnectivityApps/app/src/Codeunits/ConnectivityAppDefinitions.Codeunit.al

```diff
@@ -166,6 +167,34 @@ codeunit 20352 "Connectivity App Definitions"
         RegisterApp(AppId, AppName, AppPublisher, AppDescription, AppProviderSupportURL, AppSourceURL, AppApprovedFor, AppWorksOn, "Connectivity Apps Category"::Banking);
     end;
 
+    local procedure RegisterIQBanking()
+    var
+        AppId: Text[250];
+        AppName: Text[1024];
+        AppPublisher: Text[250];
+        AppDescription: Text[2048];
+        AppProviderSupportURL: Text[250];
+        AppSourceURL: Text[250];
+        AppWorksOn: Text;
+        AppApprovedFor: Text;
+    begin
+        /***************************************************
+            Add app 'IQ Banking' to ES
+        ***************************************************/
+
+        AppId := '80d82476-426c-4812-be05-bfcbaf777868';
+        AppName := 'IQ Banking';
+        AppPublisher := 'InnoQubit Software';
+        AppDescription := 'This app allows generate payments and check entries on your banks.';
```

Comment:
@InnoQubitAdmin Please revise the text to be more English correct. Could be something like:
"This app allows initiation of payments and import of bank entries, easily and securely."

Reply:
Updated